script/browserify.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
```

Comment:
The script name is a bit confusing. There is already a tool called `browserify` (http://browserify.org/). 
In fact, I wonder... why do you need this script, could you not just use the `browserify` tool externally to build a browser ready package? That tool bundles your script together with the dependencies like `heterarchy` and `underscore` automatically.